tests/sentry/api/endpoints/issues/test_related_issues.py

```diff
@@ -40,15 +40,12 @@ def test_same_root_related_issues(self) -> None:
         for datum in groups_data:
             self.create_group(data=datum)
 
-        response = self.get_success_response()
+        response = self.get_success_response(qs_params={"type": "same_root_cause"})
         # The UI will then make normal calls to get issues-stats
         # For instance, this URL
         # https://us.sentry.io/api/0/organizations/sentry/issues-stats/?groups=4741828952&groups=4489703641&statsPeriod=24h
         assert response.json() == {
-            "data": [
-                {"type": "same_root_cause", "data": [5], "meta": {}},
-                {"type": "trace_connected", "data": [], "meta": {}},
-            ],
+            "data": [{"type": "same_root_cause", "data": [5], "meta": {}}],
         }
 
     def test_trace_connected_errors(self) -> None:
```

Comment:
Is it worth also having a temporary test for the deprecated case which returns both types?